src/evaluate.cpp

```diff
@@ -638,8 +638,17 @@ namespace {
 
 #ifdef RACE
     if (pos.is_race())
-       b = pos.square<KING>(Us);
-    else
+    {
+        Square ksq = pos.square<KING>(Us);
+        int r = relative_rank(Up, ksq);
+        Value v = r * (r + 1) * PawnValueMg / 2;
+        Bitboard advances = in_front_bb(Us, rank_of(ksq)) & pos.attacks_from<KING>(ksq);
+        v -= (advances & ei.attackedBy[Them][ALL_PIECES]) == advances ? r * PawnValueMg : 0;
+        Bitboard blocked_squares = in_front_bb(Us, rank_of(ksq)) & ei.attackedBy[Them][ALL_PIECES];
+        v -= popcount(blocked_squares) * PawnValueMg / 5;
+        score = make_score(v, v);
+    }
+    else {
```

Comment:
In order to be able to merge upstream changes, I'm careful to use GNU style braces.